components/function-controller/README.md

```diff
@@ -0,0 +1,189 @@
+# Function Controller
+
+The Knative Function Controller is a Kubernetes controller that enables Kyma to manage Function resources. It uses
+Knative Build and Knative Serving under the hood.
+
+## Contents
```

Comment:
Contents don't match anymore, and now we have an inconsistent mixture of titles and numbers.

Reply:
yeah saw that. I have removed the numbers and contents is fixed too.

Reply:
I don't exactly agree ;)
It's not website doc, but it doesn't mean we don't have standards for it. For me, we should just have a plain installation section (without the `preparation` phase at it needs to be done anyways to use the controller). 
So just a simple flow with numbers like this:
 1. Export the variables
 2. Create serverless-system namespace
 3. Configure the controller
 4. Create the functions namespace
 5. Add function runtimes configurations
 6. Deploy the controller
 I assume all the “preparation” steps are not optional in this matter :D So let's simplify it and remove the Contents part :)

Reply:
I meant that the Contents titles don't mean the actual title anymore.